KelynPNjeri/Store-Manager
# #161041298 Add New User

#### What does this PR do?
This PR allows a Store Owner to add a new sale attendant.

#### Description of Task to be completed?
The task to be completed was to create a means by which a Store Owner could add a new sales attendant.

#### How should this be manually tested?
You can clone into my repository using the git clone `git clone https://github.com/KelynPNjeri/Store-Manager.git` to your desired location.
Go to the location to which you cloned the repository.
Change directory into the Store Manager directory using `cd Store-Manager`
Navigate to the ft-create-user-161041298 branch using the `git checkout ft-create-user-161041298` command.
Launch the `product.html` file in the html folder using your favorite web browser.
Navigate to the `My Dashboard` side navigation link to view your dashboard
Click on users and you should be able to view the users on the app plus you can also add a user if you'd like.

#### Any background context you want to provide?
None.

#### What are the relevant pivotal tracker stories?
[#161041298](https://www.pivotaltracker.com/story/show/161041298)

#### Screenshots (if appropriate)
![adduser](https://user-images.githubusercontent.com/34427048/46852714-8e74a400-ce04-11e8-8c75-b67de1c033d2.png)
